html/components/button.stories.js

```diff
@@ -23,11 +23,24 @@ always use a \`<button>\` element.
 must include a \`title=””\` attribute.
 - If a Button only includes an Icon with no text,
 alternative text must be provided.
-- If the Button has a spinner setup, the \`aria-label\`
-will be set to 'Loading' when the button is
-clicked unless you supply a custom
-string value to the \`data-sprk-spinner-aria-label\`
-attribute.
+
+##### Spinner State
+- Spinner state requires additional engineering.
+These stories only represent a spinner button after javascript intervention.
+- Add these attributes on buttons that will have a spinning state:
+  - \`class="sprk-c-Button"\` as well as any class modifiers.
+  - \`data-sprk-spinner="click"\`
+  - \`aria-live="polite"\`
+  - Other attributes seen in the story is
+  added after calling the \`setSpinning\` function.
```

Comment:
I think this sentence could be worded differently, might be a grammar thing. Maybe "is" needs to be changed to are?

Reply:
Are these other attributes added by `setSpinning` or by the implementing engineer?